pom.xml

```diff
@@ -5,22 +5,171 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-      <groupId>io.openshift.booster</groupId>
-      <artifactId>spring-boot-booster-parent</artifactId>
-      <version>1.5.13-2</version>
+      <groupId>io.openshift</groupId>
+      <artifactId>booster-parent</artifactId>
+      <version>22</version>
     </parent>
 
     <artifactId>spring-boot-cache-parent</artifactId>
     <version>1.5.13-3-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>Spring Boot - Cache Booster</name>
+    <groupId>io.openshift.booster</groupId>
```

Comment:
This should be grouped close to the `artifactId` if possible.

Reply:
That's what happens when things get automated :wink: 
I go over the PRs again and manually fix such issues

Reply:
No need for such a small issue.